modules/secret/secret_test.go

```diff
@@ -10,14 +10,22 @@ import (
 )
 
 func TestEncryptDecrypt(t *testing.T) {
-	var hex string
-	var str string
-
-	hex, _ = EncryptSecret("foo", "baz")
-	str, _ = DecryptSecret("foo", hex)
+	hex, err := EncryptSecret("foo", "baz")
+	assert.NoError(t, err)
+	str, _ := DecryptSecret("foo", hex)
 	assert.Equal(t, "baz", str)
 
-	hex, _ = EncryptSecret("bar", "baz")
+	hex, err = EncryptSecret("bar", "baz")
+	assert.NoError(t, err)
 	str, _ = DecryptSecret("foo", hex)
 	assert.NotEqual(t, "baz", str)
+
+	_, err = DecryptSecret("a", "b")
+	assert.ErrorContains(t, err, "invalid hex string")
+
+	_, err = DecryptSecret("a", "bb")
+	assert.ErrorContains(t, err, "secret key (SECRET_KEY) might be incorrect: AesDecrypt ciphertext too short")
+
+	_, err = DecryptSecret("a", "0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef")
+	assert.ErrorContains(t, err, "secret key (SECRET_KEY) might be incorrect: AesDecrypt invalid decrtyped base64 string")
```

Comment:
```suggestion
	assert.ErrorContains(t, err, "SECRET_KEY setting might be incorrect: AesDecrypt invalid decrypted base64 string")
```

Reply:
Maybe it's not SECRET_KEY

Reply:
Yes, the `might` is still there.
But apart from that I think this is a little bit less confusing as we don't repeat the same thing twice.

Reply:
There are some cases:

1. The "secret key" but not SECRET_KEY is right
2. The "secret key" but not SECRET_KEY is wrong
3. The "secret key" the same as SECRET_KEY is right
4. The "secret key" the same as SECRET_KEY is wrong

So maybe we should consider them together.


`secret key (SECRET_KEY)` means that "secret key" is possible SECRET_KEY, it's also possible not.

Reply:
That's even for me confusing, and I have a fairly good understanding of how Gitea is structured.
I think you are referring to the following, correct?
`secret key` -> the secret key to decrypt, passed as a method parameter.
`SECRET_KEY` -> the `SECRET_KEY` setting, which might be the `secret key` parameter.

Reply:
I changed the message to:

> the key (maybe SECRET_KEY?) might be incorrect: AesDecrypt invalid decrypted base64 string

Reply:
Much better.